homeassistant/scripts/influxdb_import.py

```diff
@@ -174,7 +175,7 @@ def run(script_args: List) -> int:
 
             try:
                 state = State.from_dict(event_data.get('new_state'))
-            except HomeAssistantError:
+            except (HomeAssistantError, InvalidStateError) as e:
```

Comment:
`InvalidStateError` is a subclass of `HomeAssistantError`, so this change should not be necessary at this line. Could it be necessary somewhere else in the script?

Do you have a copy of the stacktrace? If so please open a separate issue so we can have a look together.

Reply:
Hmm interesting, I must say that I did not check the exceptions inheritance, but the stack trace clearly pointed to that line with an uncaught `InvalidStateError`.
I don't have it any longer, but I might try and reproduce the run.